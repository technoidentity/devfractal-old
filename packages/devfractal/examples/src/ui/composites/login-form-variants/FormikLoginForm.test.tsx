import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { FormikLoginForm } from './FormikLoginForm'

test('Formik Login Form', () => {
  const component: ReactTestRenderer = renderer.create(<FormikLoginForm />)
  expect(component.toJSON()).toMatchSnapshot()
})
