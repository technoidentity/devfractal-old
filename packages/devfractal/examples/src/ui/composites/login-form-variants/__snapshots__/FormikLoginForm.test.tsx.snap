// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Formik Login Form 1`] = `
<form
  action="#"
  onReset={[Function]}
  onSubmit={[Function]}
>
  <fieldset>
    <label>
      Username
    </label>
    <input
      name="username"
      onBlur={[Function]}
      onChange={[Function]}
      value=""
    />
  </fieldset>
  <fieldset>
    <label>
      Password
    </label>
    <input
      name="password"
      onBlur={[Function]}
      onChange={[Function]}
      type="password"
      value=""
    />
  </fieldset>
  <fieldset>
    <button
      type="submit"
    >
       Submit 
    </button>
    <button
      type="reset"
    >
       Reset 
    </button>
  </fieldset>
</form>
`;
